Beta_Integrations/ProofpointThreatResponse/ProofpointThreatResponse.py

```diff
@@ -193,35 +192,461 @@ def delete_indicator_command():
 
 
 def test():
-    get_list(demisto.params().get('blacklist_ip'))
+    """Perform API call to check that the API is accessible.
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+    fetch_time = demisto.params().get('fetch_time')
+
+    try:
+        dateutil.parser.parse(fetch_time)
+    except ValueError as error:
+        return_error(f'There is something wrong with the fetch date. Please make sure to follow the given format. \n'
+                     f'Error: {error}')
+
+    if fetch_time[-1] != 'Z':
+        return_error('The entered fetch time is missing "Z" at its end. Please make sure to follow the given format.')
+
+    get_incidents_request(
+        {
+            'created_after': date.today(),
+            'state': 'open'
+        }
+    )
+
+
+# TRAP API
+def get_incident_field_context(incident):
```

Comment:
it's not really a good name because you manipulate the incidents, + why are we removing spaces?